crates/miden-lib/src/transaction/mod.rs

```diff
@@ -82,6 +84,12 @@ static TX_SCRIPT_MAIN: LazyLock<Program> = LazyLock::new(|| {
 pub struct TransactionKernel;
 
 impl TransactionKernel {
```

Comment:
I would also implement `to_commitment()` on `TransactionKernel`. This way, we won't have to import `SequentialCommit` every time we want to use it. It could look like so:
```rust
impl TransactionKernel {
    pub fn to_commitment(&self) -> Word {
        <Self as SequentialCommit>::to_commitment(self)
    }
}
```

Reply:
But this will only allow us to use the `to_commitment()` procedure without importing the `SequentialCommit` trait, but `to_elements()` will still require it. In that case, should we also implement something like 
```Rust
    pub fn to_elements(&self) -> Vec<Felt> {
        <Self as SequentialCommit>::to_elements(self)
    }
```

But in that case the original idea of having this trait implemented is kind of lost, because we can just implement `to_commitment()` and `to_elements()` as methods of `TransactionKernel`.